### PR TITLE
Update format for rule names

### PR DESCRIPTION
**Pull request by WilcoFiers**
- Remove backticks
- Remove single quotes
- Always capitalize

We have a number of places where rendering back ticks in rule names has proven difficult. We've previously stripped out the back ticks, but this ends up looking kinda strange, with some rule names capitalised and others not for no obvious reason. 

Need for Call for Review: 1 week

---

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/audio-or-video-avoids-automatically-playing-audio-80f0bf.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 id: 80f0bf
-name: '`audio` or `video` avoids automatically playing audio'
+name: Audio or video avoids automatically playing audio
```
Comment (kengdoj) [inline _rules/audio-or-video-avoids-automatically-playing-audio-80f0bf.md:3]:
```suggestion
name: Audio element or video element avoids automatically playing audio
```

File: _rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 id: aaa1bf
-name: '`Audio` or `video` that plays automatically has no audio that lasts more than 3 seconds'
+name: Audio or video that plays automatically has no audio that lasts more than 3 seconds
```
Comment (kengdoj) [inline _rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md:3]:
```suggestion
name: Audio element or video element that plays automatically has no audio that lasts more than 3 seconds
```

File: _rules/auto-play-audio-has-control-mechanism-4c31df.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 id: 4c31df
-name: '`audio` or `video` that plays automatically has a control mechanism'
+name: Audio or video that plays automatically has a control mechanism
```
Comment (kengdoj) [inline _rules/auto-play-audio-has-control-mechanism-4c31df.md:3]:
```suggestion
name: Audio element or video element that plays automatically has a control mechanism
```

**Review by kengdoj — CHANGES_REQUESTED** [review]
added element after audio and video in some of the rules

**Review by kengdoj — APPROVED** [review]

File: _rules/meta-refresh-no-delay-bc659a.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 id: bc659a
-name: '`meta` element has no refresh delay'
+name: Meta element has no refresh delay'
```
Comment (carlosapaduarte) [inline _rules/meta-refresh-no-delay-bc659a.md:3]:
```suggestion
name: Meta element has no refresh delay
```

File: _rules/table-headers-attribute-refer-to-data-cells-a25f45.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 id: a25f45
-name: '`headers` attribute specified on a cell refers to cells in the same `table` element'
+name: Headers attribute specified on a cell refers to cells in the same table element'
```
Comment (carlosapaduarte) [inline _rules/table-headers-attribute-refer-to-data-cells-a25f45.md:3]:
```suggestion
name: Headers attribute specified on a cell refers to cells in the same table element
```

**Review by carlosapaduarte — CHANGES_REQUESTED** [review]
Missed a couple

**Review by Jym77 — APPROVED** [review]

**Review by colabottles — APPROVED** [review]

**Pull request merged**